Docs: align self-driving positioning (apps → tools) (#17867)
PostHog/posthog.com: contents/docs/self-driving/web.mdx

import { CallToAction } from 'components/CallToAction'
import { AppsList } from 'components/Docs/AppsList'

> The [PostHog web app](https://app.posthog.com) is the PostHog you already know, now a surface for self-driving. It's the full product in your browser, and it ties the whole loop together: your data lives here, your tools run here, and the inbox where you review the work agents do lives here too.

Author: ```suggestion
[PostHog Web](https://app.posthog.com) is the PostHog you already know, now a surface for self-driving. It's the full product in your browser, and it ties the whole loop together: your data lives here, your tools run here, and the inbox where you review the work agents do lives here too.
```

> It's one of the surfaces you work from, alongside [Slack](/docs/slack) and [MCP](/docs/model-context-protocol). Slack is for summoning agents in the flow of your day. The web app is for going deep: exploring your data, reviewing proposed work, and steering what self-driving does.

Author: ```suggestion
It's one of the products you use, alongside [Slack](/docs/slack) and [MCP](/docs/model-context-protocol). Slack is for summoning agents in the flow of your day. The web app is for going deep: exploring your data, reviewing proposed work, and steering what self-driving does.
```

See the [inbox docs](/docs/self-driving/inbox) for how it works.

## Your tools

> Every PostHog tool — product analytics, session replay, feature flags, and the rest — is something you use in the web app:

Author: ```suggestion
Every PostHog tool — product analytics, session replay, feature flags, and the rest — is something you use in PostHog Web:
```
